salt/modules/cassandra_cql.py

```diff
@@ -0,0 +1,585 @@
+
+        salt 'minion1' cassandra.info
+
+        salt 'minion1' cassandra.info contact_points=minion1
+    '''
+
+    query = 'select cluster_name, \
+             data_center, \
+             partitioner, \
+             host_id, \
+             rack, \
+             release_version, \
+             cql_version, \
+             schema_version, \
+             thrift_version from system.local limit 1'
+    ret = _query(query, contact_points, port, cql_user, cql_pass)
+    if ret:
+        return ret[0]
+
+
+def list_keyspaces(contact_points=None, port=None, cql_user=None, cql_pass=None):
+    '''
+    List keyspaces in a Cassandra cluster.
+
+    :param contact_points: The Cassandra cluster addresses, can either be a string or a list of IPs.
+    :type  contact_points: str | list[str]
+    :param cql_user:       The Cassandra user if authentication is turned on.
+    :type  cql_user:       str
+    :param cql_pass:       The Cassandra user password if authentication is turned on.
+    :type  cql_pass:       str
+    :param port:           The Cassandra cluster port, defaults to None.
+    :type  port:           int
+    :return:               The keyspaces in this Cassandra cluster.
+    :rtype:                list[dict]
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt 'minion1' cassandra.list_keyspaces
+
+        salt 'minion1' cassandra.list_keyspaces contact_points=minion1 port=9000
+    '''
+    query = 'select keyspace_name from system.schema_keyspaces'
+    ret = _query(query, contact_points, port, cql_user, cql_pass)
+    if ret:
+        return ret
+    else:
+        return False
+
+
+def list_column_families(keyspace=None, contact_points=None, port=None, cql_user=None, cql_pass=None):
+    '''
+    List column families in a Cassandra cluster for all keyspaces or just the provided one.
+
+    :param keyspace        The keyspace to provide the column families for, optional.
+    :type  keyspace:       str
+    :param contact_points: The Cassandra cluster addresses, can either be a string or a list of IPs.
+    :type  contact_points: str | list[str]
+    :param cql_user:       The Cassandra user if authentication is turned on.
+    :type  cql_user:       str
+    :param cql_pass:       The Cassandra user password if authentication is turned on.
+    :type  cql_pass:       str
+    :param port:           The Cassandra cluster port, defaults to None.
+    :type  port:           int
+    :return:               The column families in this Cassandra cluster.
+    :rtype:                list[dict]
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt 'minion1' cassandra.list_column_families
+
+        salt 'minion1' cassandra.list_column_families contact_points=minion1
+
+        salt 'minion1' cassandra.list_column_families keyspace=system
+    '''
+    query = 'select columnfamily_name from system.schema_columnfamilies'
+    if keyspace:
+        where_clause = 'where keyspace_name = \'%s\'' % keyspace
```

Comment:
Salt is using formatting instead. Also you can use double-quotes to avoid unnecessary backslashes. So this should be written the following way:
```python
where_clause = "where keyspace_name = '{0}'".format(keyspace)
```